### DISCUSSION TITLE
Snyk security is warning of DDoS vulnerability on package.

### DISCUSSION THREAD

**Discussion by sewellstephens** [expressjs/multer#1253] [General]
Hi, I'm not at all an expert in cybersecurity nor am I a security researcher, but Snyk security and NPM is warning of critical DDoS vulnerability attached to this package. According to NPM vuln scanner, it seems to be linked to busboy dependency in package.json. 

### Heres a screenshot:
<img width="657" alt="dicer vuln" src="https://github.com/expressjs/multer/assets/41175080/458f7eb0-5c1f-4a4f-9d6b-36bbe9148b27">

**Comment by UlisesGascon** [comment]
I will close this in favor of https://github.com/expressjs/multer/issues/1254. Please @sewellstephens in the future try to use a single channel (issues or discussions) to keep the thread in a single place :wink:
